build.zig

```diff
@@ -403,6 +403,11 @@ fn getDependencies(b: *std.Build, target: std.Build.ResolvedTarget, optimize: st
         const flags_dep = Dependency{ .name = "flags", .module = flags.module("flags") };
         try dependencies.append(flags_dep);
 
+        // Known-Folders
```

Comment:
Interesting; I'll take a look at this repo.  I was only ever targeting Linux, but if this helps support other platforms, seems useful.